pkgs/development/libraries/libheif/default.nix

```diff
@@ -1,14 +1,15 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, autoreconfHook
+, cmake
 , pkg-config
 , dav1d
 , rav1e
 , libde265
 , x265
 , libpng
 , libjpeg
+, libwebp
```

Comment:
Why is libwebp needed, I can't see anything upstream about it?

Reply:
I think I'd originally had a build error without it however at least now, I don't.

Removed in https://github.com/NixOS/nixpkgs/pull/254708/commits/de0104d374dd04034686b32a29aaaf004b3ef3bc